File: week-04/day-02/GreenFox_org/Student.py
from Person import Person

class Student(Person):
    def __init__(self, previous_organization = "The School of Life", skipped_days = 0):
        super().__init__()
        self.previous_organization = previous_organization
        self.skipped_days = skipped_days


    def get_goal(self):
         print("Be a junior software developer.")


    def introduce(self):
        print('Hi, I\'m {}, a {} year old {} from {} who skipped {} days from the course already.'.format(self.name,self.age, self.gender, self.previous_organization, self. skipped_days))


    def skip_days(self, number_of_days = 0):
        skip_days += number_of_days

File: Csharp-course/week-03/day-05/GreenFox/GreenFox/Student.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace GreenFox
{
    class Student : Person
    {
        public string previousOrganization;
        public int skippedDays;

        public Student(string _name, int _age, string _gender, string prevOrg)
        {
            this.name = _name;
            this.age = _age;
            this.gender = _gender;
            this.previousOrganization = prevOrg;
            this.skippedDays = 0;
        }

        public Student()
        {
            name = "Jane Doe";
            age = 30;
            gender = "female";
            previousOrganization = "The School of Life";
            skippedDays = 0;
        }

        public override void GetGoal()
        {
            Console.WriteLine("Hi, I'm name, a {0} year old {1} {2} from {3} who skipped {4} days from the course already.",this.name,this.age,this.gender,this.previousOrganization, this.skippedDays);
        }

        public int SkipDays(int number)
        {
            return this.skippedDays += number;
        }
    }
}
